styx-common/src/main/java/com/spotify/styx/state/RunState.java

```diff
@@ -184,6 +184,22 @@ public RunState created(WorkflowInstance workflowInstance, String executionId, S
     }
 
     @Override
+    public RunState info(WorkflowInstance workflowInstance, String message) {
```

Comment:
It would be useful to also pass in the message level with the info event, instead of always being at warning level.